lambdas/src/apis/collections/controllers/collections.ts

```diff
@@ -30,10 +31,67 @@ export async function getStandardErc721(fetcher: SmartContentServerFetcher, req:
             res.status(404).send()
         }
     } catch(e) {
+        console.log(e)
         res.status(500).send(e.messsge);
     }
 }
 
+export async function contentsImage(fetcher: SmartContentServerFetcher, req: Request, res: Response) {
+    // Method: GET
+    // Path: /contents/:contract/:option/image
+    const { contract, option } = req.params;
+
+    await internalContents(fetcher, res, contract, option, wearableMetadata => wearableMetadata.image)
+}
+
+export async function contentsThumbnail(fetcher: SmartContentServerFetcher, req: Request, res: Response) {
+    // Method: GET
+    // Path: /contents/:contract/:option/thumbnail
+    const { contract, option } = req.params;
+
+    await internalContents(fetcher, res, contract, option, wearableMetadata => wearableMetadata.thumbnail)
+}
+
+async function internalContents(fetcher: SmartContentServerFetcher, res: Response, contract: string, option: string, selector: (WearableMetadata: WearableMetadata) => string |undefined) {
+    try {
+        let contentBuffer: Buffer | undefined = undefined
+        const entities:Entity[] = await fetcher.fetchJsonFromContentServer(`/entities/wearable?pointer=${contract}-${option}`)
+        if (entities && entities.length > 0 && entities[0].metadata) {
+            const wearableMetadata: WearableMetadata = entities[0].metadata
+            const relativeContentUrl = createRelativeContentUrl(fetcher, entities[0], selector(wearableMetadata))
+            if (relativeContentUrl) {
+                contentBuffer = await fetcher.fetchBufferFromContentServer(relativeContentUrl) // TODO: fetch a stream instead of a Buffer
```

Comment:
Could you create a github issue for this TODO?

Reply:
Sure!